discovery_climate_music_analysis/notebooks/2_chart_analysis.py

```diff
@@ -0,0 +1,198 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     comment_magics: true
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.14.5
+#   kernelspec:
+#     display_name: discovery_climate_music_analysis
+#     language: python
+#     name: discovery_climate_music_analysis
+# ---
+
+# %% [markdown]
+# # Chart Analysis
+
+# %%
+import numpy as np
+import pandas as pd
+import altair as alt
+import configparser
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+from pprint import pprint
+from dateutil.parser import parse
+
+from discovery_climate_music_analysis.getters.collect_songs import create_dataframe
+from discovery_climate_music_analysis.getters.collect_chart_songs import load_s3_data
+from jacc_hammer.fuzzy_hash import Cos_config, Fuzzy_config, match_names_stream
+
+USERNAME = "addie234"
+PLAYLIST_ID = "spotify:playlist:4EhBYkl4jtI2POrkUxU6Ul"
+
+
+bucket_name = "discovery-hub-open-data"
+
+# %%
+# read chart data
+all_top_charts = load_s3_data(
+    bucket_name, "climate_change_environment_songs/chart_songs.csv"
+)
```

Comment:
Hmm, this one returned an error. I did check that the file is on s3, and I double checked that permissions are such that it can be accessed even without aws credentials, so not sure why the error...
```
FileNotFoundError: discovery-hub-open-data/climate_change_environment_songs/chart_songs.csv
```

Reply:
For now, I worked around this by downloading the file and reading it using pandas. Perhaps you have a getter that can load either from s3 or from local disk, depending on user preference?